video/out/vulkan/context.c

```diff
@@ -238,6 +239,7 @@ pl_vulkan mppl_create_vulkan(struct vulkan_opts *opts,
         .num_opt_extensions = MP_ARRAY_SIZE(opt_extensions),
         .features = &features,
         .device_name = is_uuid ? NULL : opts->device,
+        .allow_software = opts->allow_sw,
```

Comment:
Put it in the same place as in the `pl_vulkan_params`, meaning before `async_transfer`